docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/private-api-is-extended.md

```diff
@@ -14,6 +14,8 @@ related:
     link: docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/private-api-method-is-overridden-on-the-project-level.html
 ---
 
+Private API is extended. The page describes why entities on the project side have to avoid private api extending. The project has to follow the rule to be able to correctly receive Spryker autoupdates.
```

Comment:
And as I described before it's better to start with successful scenario, so something like:

```
Private API entities shouldn't be extended on the project side. This will allow you to improve the process of semi-automated delivery of Spryker module updates.
```

And don't use spelling ```autoupdates```. We are not allowed to name the process as "automatical".

Reply:
++ 

Replaced with the suggested structure